gradle.properties

```diff
@@ -18,4 +18,4 @@ kotlin.code.style=official
 
 release_version=5.10.0
 
-docker_image_name=
\ No newline at end of file
+docker_image_name=
```

Comment:
You can publish an empty commit to achieve the same results:
```
git commit --allow-empty -m "Retrigger build"
```
However, this looks strange and probably instead of having the while-card version you should specify the exact one if you have to make a commit to trigger the build anyway

Reply:
I know about this approach, but it doesn't work for some repo